src/Telemetry/Microsoft.DotNet.AzureDevOpsTimelineTests/MockBuildLogScraper.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.AzureDevOpsTimeline.Tests
+{
+    public class MockBuildLogScraper : IBuildLogScraper
```

Comment:
Are we using `Moq` here?

You could remove this class and do something like:

```csharp
var mock = new Mock<IBuildLogScraper>();
mock.SetReturnsDefault(Task.FromResult(""));
```

Reply:
This class wasn't used, removed